core/src/lib.rs

```diff
@@ -1,3 +1,11 @@
+pub mod collections;
+pub mod conn;
+pub mod databases;
+pub mod fields;
+pub mod query;
+pub mod stmt;
```

Comment:
similar to my comment below re: the `err` module, I think we should make these modules private, and `pub use` the items we want to export from each